app/defines/random.h

```diff
@@ -0,0 +1,53 @@
+/**********************************************************
+ * random numbers library - based on 32bit LFSR algorithm
+**********************************************************/
+
+#ifndef RANDOM_H
+#define RANDOM_H
+
+unsigned int lfsr_seed;
```

Comment:
I think we should we init the LFSR seed to some value

Reply:
changed to 1